packages/fstar/fstar.2022.01.15/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "Verification system for effectful programs"
+maintainer: "taramana@microsoft.com"
+authors: [
+  "Nik Swamy <nswamy@microsoft.com>"
+  "Jonathan Protzenko <protz@microsoft.com>"
+  "Tahina Ramananandro <taramana@microsoft.com>"
+]
+license: "Apache-2.0"
+homepage: "http://fstar-lang.org"
+bug-reports: "https://github.com/FStarLang/FStar/issues"
+depends: [
+  "ocaml" {>= "4.04.1" & < "4.13.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.04.1" & < "4.13.0"}
```
Is there any particular reason why this package is not compatible with 4.13?

Reply:
Replying to my own question: there doesn’t seem to be any reason.
I’ve sent a patch upstream: https://github.com/FStarLang/FStar/pull/2434

Reply:
Thanks!